javascript.js

```diff
@@ -0,0 +1,49 @@
+let library = [];
+
+function getInput() {
+  const book = {};
+  book.title = document.getElementById('bookTitle').value;
+  book.author = document.getElementById('bookAuthor').value;
+  return book;
+}
+
+function removeBook(title) {
+  const book = document.getElementById(title);
+  book.remove();
+  library = library.filter((bookObj) => bookObj.title !== title);
+  localStorage.setItem('library', JSON.stringify(library));
```

Comment:
- [ ] Hi you did a great job but taking the title as the identifier is not the best method because after removing a book, all the books with the same name are removed from storage. ill recommend adding an `id` to your object which will hold a unique value per book
i.e a better method could be to use the date as the generated id `Date.now()`

Reply:
Hello, @m05e5 I need clarification on what you mean here. Each title as an id holds its unique instance. removing one book can not affect another book even though their share the same title. I tested it and it is proven. I am open to your clarification.

Reply:
Fixed, thank you for feedback.